db/migrate/20220811014712_create_user.rb

```diff
@@ -0,0 +1,13 @@
+class CreateUser < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :photo
+      t.string :bio
```

Comment:
- [ ] Kindly use `text` as a datatype for the bio because a string only limit to 255 characters